### DISCUSSION TITLE
Add tesseract_ros_examples to the docs

### DISCUSSION THREAD

**Discussion by mohamedsayed18** [tesseract-robotics/tesseract_docs#7] [Ideas]
I wrote a readme explaining the puzzle_auxiliray example [here](https://github.com/ros-industrial-consortium/tesseract_ros/pull/113)
and prepared a new one for the glass_upright_example.
Should we move these tutorials here in the docs or in the tesseract_ros repo?

**Comment by Levi-Armstrong** [comment]
Yea, I think that will be the plan.
